homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,44 @@
+const tamagochi = {
+    name: 'Tamagochi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    noEating() {
+        tamagochi.healthLevels -= 1;
+        tamagochi.happyLevels -= 1;
+        tamagochi.foodLevels -= 1;
+
+        const tamagochiStates = `Здоров'я  в ${tamagochi.name} - ${tamagochi.healthLevels} Щастя - ${tamagochi.happyLevels} Ситість - ${tamagochi.foodLevels}`;
+
+        const tamagochiDiedFirst = tamagochi.healthLevels === 0 || tamagochi.happyLevels === 0;
+        const tamagochiDiedSecond = tamagochi.foodLevels === 0;
+
+        if (tamagochiDiedFirst === true && tamagochiDiedSecond === true) {
+            console.log(`${tamagochi.name} вмер... ${tamagochiStates}`);
+        } else if (tamagochi.foodLevels === 1) {
+            console.log(`Покорми, ${tamagochi.name}, бо вiн скоро вмре... ${tamagochiStates}`);
+        } else {
+            console.log(`Сьогодні ${tamagochi.name} нічого не вполював... ${tamagochiStates}`);
+        }
```

Comment:
In general, you can also unify and move this logic into a separate method. If so, you will be able to create many methods which can change the characteristics of the Tamagotchi and check if it's alive just by one method.